"Ninja flash" swallows user prompts

I have a problem using 'ninja flash' when I have multiple boards of the same type plugged in (multiple PyOCD boards, etc.). The issue is that the flasher scripts prompt the user for which board to flash from a list, but ninja swallows the prompt, so the command hangs forever.

This is a regression from "make flash".

I think the issue is CMake, since ninja uses CMake under the hood to invoke the flasher script.

```
$ ninja flash
[1/82] Generating always_rebuild
Building for board nrf51_pca10028
[2/2] Flashing nrf51_pca10028
FAILED: zephyr/cmake/flash/CMakeFiles/flash
cd /home/carles/src/zephyr/samples/bluetooth/beacon/b && /usr/bin/cmake -E env /usr/bin/python3 /home/carles/src/zephyr/scripts/support/zephyr_flash_debug.py nrfjprog flash --board-dir=/home/carles/src/zephyr/boards/arm/nrf51_pca10028 --kernel-elf=/home/carles/src/zephyr/samples/bluetooth/beacon/b/zephyr/zephyr.elf --kernel-hex=/home/carles/src/zephyr/samples/bluetooth/beacon/b/zephyr/zephyr.hex --kernel-bin=/home/carles/src/zephyr/samples/bluetooth/beacon/b/zephyr/zephyr.bin --gdb=/home/carles/prog/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gdb --openocd=/home/carles/prog/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/openocd --openocd-search=/home/carles/prog/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/share/openocd/scripts --nrf-family=NRF51
There are multiple boards connected.
1. 681089210
2. 681726921
Please select one with desired serial number (1-2): Error: EOF when reading a line
(Re-run as "/home/carles/src/zephyr/scripts/support/zephyr_flash_debug.py --verbose nrfjprog ..." or set CMAKE_VERBOSE_MAKEFILE for a stack trace.)
ninja: build stopped: subcommand failed.
```

--- a/cmake/flash/CMakeLists.txt
+++ b/cmake/flash/CMakeLists.txt
@@ -70,5 +70,6 @@ foreach(target flash debug debugserver)
     ${cmd}
     COMMENT
     ${comment}
+    USES_TERMINAL
     )
 endforeach()

